### PR TITLE
Update the way of decaying the learning rate

### PR DESCRIPTION
**Pull request by philokey**
Avoid the problem of training speed become slower iteration by iteration mentioned in this [issue](https://github.com/smallcorgi/Faster-RCNN_TF/issues/10)

### CODE REVIEW

**Comment by ICapalija** [comment]
In your code learning rate is going to be changed just once, but don't you think that a better approach is to allow multiple corrections of learning rate. Please check my commit c018d838d3b3ee2a4770f60fdfb44e754e8290bd

**Comment by philokey** [comment]
No it will decay every `cfg.TRAIN.STEPSIZE` steps with a base of 0.1:
You can get more details [here](https://www.tensorflow.org/versions/r0.11/api_docs/python/train.html#exponential_decay)

**Comment by ICapalija** [comment]
Yes, you are right. I have been testing the wrong code :(

**Comment by hzy46** [comment]
This commit is right. The training becomes very slower on my machine.
